Source/NuGetTemplate/azure-pipelines.yml

```diff
@@ -0,0 +1,122 @@
+# Azure Pipelines documentation https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - '*'
+
+variables:
+  # Set the DOTNET_SKIP_FIRST_TIME_EXPERIENCE environment variable to stop wasting time caching packages
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  # Disable sending usage data to Microsoft
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+
+stages:
+- stage: Build
+  jobs:
+  - job: Build
+    strategy:
+      matrix:
+        Linux:
+          matrixName: Ubuntu
+          vmImageName: ubuntu-latest
+        Mac:
+          matrixName: Mac
+          vmImageName: macos-latest
+        Windows:
+          matrixName: Windows
+          vmImageName: windows-latest
+    pool:
+      vmImage: $(vmImageName)
+    timeoutInMinutes: 10
+    steps:
+    - checkout: self
+      lfs: true
+    - task: UseDotNet@2
+      displayName: 'Install .NET Core SDK'
+      inputs:
+        packageType: 'sdk'
+        version: '3.1.101'
+    - script: 'dotnet tool install --global Cake.Tool'
```

Comment:
Why do this instead of shipping a tool manifest and doing `dotnet tool restore`?

Reply:
Yeah maybe. Looking into it.

Reply:
All my repos now use a `dotnet tool manifest`. Turns out it's slightly better than installing a global tool.